frontend/components/ClubEditPage/AdvisorCard.tsx

```diff
@@ -14,9 +17,35 @@ import BaseCard from './BaseCard'
 
 type Props = {
   club: Club
+  validateAdvisors?: (valid: boolean) => void
 }
 
-export default function AdvisorCard({ club }: Props): ReactElement {
+const RequireText = styled.p`
+  color: ${RED};
+  margin-top: 1rem;
+`
+
+export default function AdvisorCard({
+  club,
+  validateAdvisors,
+}: Props): ReactElement {
+  const [advisorsCount, setAdvisorsCount] = useState<number>(
```

Comment:
@alnasir7 You should remove this state variable if you're not using it anywhere else.

Reply:
It could be nice to only display the "At least one public contact is required*" text only if there is an insufficient number of advisors, to make it more apparent when the condition is met.